src/codecs/mozjpeg/options.tsx

```diff
@@ -0,0 +1,27 @@
+import { h } from 'preact';
+import { CodecOptions } from '../../components/app';
+
+type Props = {
+  options: CodecOptions,
+  updateOption(e: Event): void,
```

Comment:
For event-driven callbacks, I tend to name them `onWhatever`. Is this a good convention? +@surma.